node.js

```diff
@@ -258,5 +265,24 @@ module.exports = {
   clearCaches: function clearCaches () {
     filenessCache = { }
     configCache = { }
+  },
+
+  testCanIUse: function testCanIUse (latestVersion) {
+    var now = Date.now()
+    var digitsDiff = String(now).length - String(latestVersion).length
+    if (digitsDiff) {
+      latestVersion = padToEnd(latestVersion, digitsDiff)
+    }
+    var diffDates = now - latestVersion
+    diffDates = diffDates / 1000 / 60 / 60
+    var daysFromUpdate = Math.floor(diffDates / 24)
+    if (daysFromUpdate > DAYS_TO_UPDATE_CANIUSE) {
+      var pathToYarnLock = path.normalize('./yarn.lock')
+      var isYarnInstalled = isFile(pathToYarnLock)
+      var packageManagerStr = isYarnInstalled ? 'yarn upgrage' : 'npm update'
```

Comment:
`npm update` will update many packages and could break an application. A user needs to update only `caniuse-lite` and `browserslist`.

Reply:
seems you missed this part of message at the end ```'\x1b[0m\'caniuse-lite\' is outdated.```

Reply:
Yeap. The code is too complex :(. Make it clear by `Run ' + packageManager + ' update caniuse-lite browserslist '`